x/committee/keeper/querier.go

```diff
@@ -166,39 +165,18 @@ func queryVote(ctx sdk.Context, path []string, req abci.RequestQuery, keeper Kee
 //				Tally
 // ------------------------------------------
 
-func queryTally(ctx sdk.Context, path []string, req abci.RequestQuery, keeper Keeper) ([]byte, error) {
+func queryTally(ctx sdk.Context, req abci.RequestQuery, keeper Keeper, legacyQuerierCdc *codec.LegacyAmino) ([]byte, error) {
 	var params types.QueryProposalParams
-	err := keeper.cdc.UnmarshalJSON(req.Data, &params)
+	err := legacyQuerierCdc.UnmarshalJSON(req.Data, &params)
 	if err != nil {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrJSONUnmarshal, err.Error())
 	}
-
-	proposal, found := keeper.GetProposal(ctx, params.ProposalID)
+	tally, found := keeper.GetProposalTallyResponse(ctx, params.ProposalID)
 	if !found {
-		return nil, sdkerrors.Wrapf(types.ErrUnknownProposal, "%d", params.ProposalID)
-	}
-
-	committee, found := keeper.GetCommittee(ctx, proposal.CommitteeID)
-	if !found {
-		return nil, sdkerrors.Wrapf(types.ErrUnknownCommittee, "%d", proposal.CommitteeID)
-	}
-
-	var pollingStatus types.ProposalPollingStatus
```

Comment:
`ProposalPollingStatus` has the exact same fields as `QueryTallyResponse` so I believe this legacy rest endpoint should return the same data